STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -402,8 +405,8 @@ initiated on the IdO-CA side.
 
 The Order object created by the IdO:
```

Comment:
To clarify (because we discussed earlier "The Order object that is created on the IdO"), I suggest to reword:

> The Order object created by the IdO in its request from the ACME Server:

Note that the same change is needed in the non-STAR section below.

Reply:
Should we not `s/from/to/` in _"The Order object created by the IdO in its request **from** the ACME Server:"_?

Reply:
See if 4cb0cd6 is what you meant.

Reply:
> 👍 but should we not `s/from/to/` in _"The Order object created by the IdO in its request **from** the ACME Server:"_?

I think both are legitimate, and therefore I'd leave that to the RFC Editor.